core/src/main/java/io/questdb/griffin/FunctionParser.java

```diff
@@ -619,9 +710,9 @@ private Function createFunction(
                 return checkAndCreateFunction(factory, args, argPositions, node, configuration);
             }
 
-            if (candidateDescriptor == null) {
-                candidateDescriptor = descriptor;
-            }
+//            if (candidateDescriptor == null) {
```

Comment:
Looks like a leftover.

Reply:
removed in https://github.com/questdb/questdb/pull/4632/commits/a27f01594461930d022aac04163dce85937301b9